addons/web/static/src/core/pager/pager.xml

```diff
@@ -2,13 +2,13 @@
 <templates xml:space="preserve">
 
     <t t-name="web.Pager" owl="1">
-        <nav class="o_pager" aria-label="Pager">
+        <nav class="o_pager align-items-center d-flex" aria-label="Pager">
             <span class="o_pager_counter mr-2" t-on-click.stop="">
```

Comment:
Can we remove mr-2 and use gap instead?

Reply:
Unfortunately BTS 4 does not support `gap` but this margin could be replaced by `.gap-2` after the migration to BTS5.

Reply:
Yes I know and it's why we use gap in scss to be able to easily find them to then modify once again the xml file with the correct BT5 class... So, as it's not the first time we need this class, I propose to backport it to do the job properly.